src/main.js

```diff
@@ -23,6 +23,7 @@ global.data.stats = global.data.stats || {
   memoryFree: 0,
   heapFree: 0,
 };
+global.maliciousNpcUsernames = ['Invader', 'Source Keeper'];
```

Comment:
Please move this into the config instead.

Reply:
@TooAngel done.